lib/rules/method.js

```diff
@@ -71,6 +71,21 @@ function checkCallExpression(ruleHelper, callExpr, node) {
         ruleHelper.checkMethod(callExpr);
         break;
 
+
+    case "SequenceExpression": {
+        // the return value of a SequenceExpression is the last expression.
+        // So, we create a new mock CallExpression with the actually called
+        // ... expression as the callee node and pass it to checkMethod()
+
+        /* eslint-disable no-case-declarations */
+        const newCallExpr = Object.assign({}, callExpr);
+        const idx = node.expressions.length - 1;
+        const called = node.expressions[idx];
+        newCallExpr.callee = called;
+        ruleHelper.checkMethod(newCallExpr);
+        break;
+        /* eslint-enable no-case-declarations */
+    }
     // those are fine:
```

Comment:
It looks that eslint would like an empty line between this comment and the line right before it:
- https://travis-ci.org/github/mozilla/eslint-plugin-no-unsanitized/jobs/663528026#L214